src/components/shared/thread/ActivityGraph.js

```diff
@@ -90,19 +80,10 @@ class ThreadActivityGraph extends React.PureComponent<Props, State> {
     });
   };
 
-  _takeCanvasRef = (canvas: HTMLCanvasElement | null) => {
-    this._canvas = canvas;
+  _setFillsQuerier = (fillsQuerier: ActivityFillGraphQuerier) => {
```

Comment:
This data flow is unconventional for React and Redux. There are a few places where we do this, but it would be nice to not have to do this. Perhaps the logic needing this dependency can be done in the canvas component. If we do need to go this route, then we'll need good code documentation justifying the use.

Edit: After reading this a bit, I see why this is happening. As I stated elsewhere, it would be good to document the reasoning here.